spec/controllers/periodic_expenses_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe PeriodicExpensesController, type: :controller do
```

Comment:
What about having an integration test instead of a controller one? Have you thought about it?